charts/customer-center/Chart.yaml

```
@@ -14,3 +14,10 @@ maintainers:
   - name: adfinis
     email: support@adfinis.com
     url: https://adfinis.com
+annotations:
+  artifacthub.io/changes: |
+    - kind: changed
+      description: "update customer-center from v3.2.4 to v3.2.5"
+      links:
+        - name: GitHub Commit
+          url: https://github.com/adfinis/customer-center/commit/c919df22a52a2fecff633935dbd08a8b03057918
```

Wouldn't it be better to link to the GitHub release?

```suggestion
        - name: GitHub Release
          url: https://github.com/adfinis/customer-center/releases/tag/v3.2.5
```

---

Didn't think about that. Makes totally sense. Thanks :)